ArduCopter/AP_Arming.cpp

```diff
@@ -191,6 +191,7 @@ bool AP_Arming_Copter::board_voltage_checks(bool display_failure)
 {
     #if CONFIG_HAL_BOARD != HAL_BOARD_VRBRAIN
     #ifndef CONFIG_ARCH_BOARD_PX4FMU_V1
+    #ifndef CONFIG_ARCH_BOARD_AEROFC_V1
```

Comment:
while not strictly required for this PR, adding a HAL_HAVE_BOARD_VOLTAGE would be nice